### PR TITLE
feat(ontology): update cardinality in resource class (DSP-1266)

### PR DESCRIPTION
**Pull request by kilchenmann**
resolves DSP-1266

It works with new created ontologies. Most of the test ontologies doesn't have a `lastModificationDate` and they can't be edited. 
Here are two screen records about "How to update cardinality":

https://user-images.githubusercontent.com/4253580/107492595-f0887400-6b8c-11eb-9367-0f571dec7c3c.mp4

https://user-images.githubusercontent.com/4253580/107492581-ecf4ed00-6b8c-11eb-8295-ff62b2b2abf6.mp4

The wording will be done in a separate task.

### CODE REVIEW

File: src/app/project/ontology/property-form/property-form.component.ts [hunk 1]
```diff
@@ -164,8 +178,9 @@ export class PropertyFormComponent implements OnInit {
      * @returns Filtered list of options
      */
     filter(list: AutocompleteItem[], label: string) {
+        console.log(list)
```
Comment (flavens) [inline src/app/project/ontology/property-form/property-form.component.ts:181]: this console.log must be removed

File: src/app/project/ontology/resource-class-form/resource-class-form.service.ts [hunk 2]
```diff
@@ -83,7 +83,14 @@ export class ResourceClassForm {
 
     constructor(resourceClass: ResourceClass) {
         if (resourceClass.properties) {
-            this.properties.setValue([resourceClass.properties]);
+            let i = 0;
+            this.properties.setControl
+            resourceClass.properties.forEach(prop => {
+                this.properties[i] = new FormControl(prop);
+                // this.properties[i].setValue(prop);
```
Comment (flavens) [inline src/app/project/ontology/resource-class-form/resource-class-form.service.ts:90]: should we keep these commented lines?

**Review by flavens — COMMENTED** [review]
some tiny things

File: src/app/project/ontology/property-form/property-form.component.ts (hunk 1 above)
Comment (kilchenmann) [reply src/app/project/ontology/property-form/property-form.component.ts:181]: done in d74a77d

File: src/app/project/ontology/ontology.component.html [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <div *ngIf="projectAdmin" class="desktop-only">
 
-    <p class="note warning center">This is a first version of the ontology editor. Some features may not work as intended.</p>
+    <p class="note warning center">This is a first version of the ontology editor. Some features may not work as
```
[the remark this answers is not in the document]
Comment (kilchenmann) [reply src/app/project/ontology/ontology.component.html:3]: done in 4df6c48

File: src/app/project/ontology/resource-class-form/resource-class-form.service.ts (hunk 2 above)
Comment (kilchenmann) [reply src/app/project/ontology/resource-class-form/resource-class-form.service.ts]: done in de4ac1f

**Review by subotic — APPROVED** [review]

**Pull request merged**